docs/users_guide/ocean/test_groups/drying_slope.rst

```diff
@@ -0,0 +1,85 @@
+.. _ocean_drying_slope:
+
+drying_slope
+=============
+
+The ``ocean/drying_slope`` test group reproducs the tidal channel flow test case
+from `Warner et al. (2013) <http://dx.doi.org/10.1016/j.cageo.2013.05.004>`_
+which was originally presented in
+`Oey (2005) <https://doi.org/10.1016/j.ocemod.2004.06.002>`_.
+
+The domain is 2-d with a channel sloping in the x-dimension and periodic in the
+y-dimension. The channel length is set to 25 km and slopes with an aspect ratio
+of 0.1. The channel width is 6km for all simulations. The channel is initialized
+as dry with a minimum layer thickness. The water level at the seaward end is
+oscillated sinusoidally with a period of 0.5 days as
+
+.. math::
+   \zeta_{east} = 10.0 \sin(\pi/0.5t) - 10.0
+
+.. image:: images/drying_slope_forcing.png
+   :width: 500 px
+   :align: center
+
+Temperature and salinity are constant throughout the domain with values of 20
+degC and 35 PSU, respectively. Two different linear drag bottom roughness
+factors of 0.0025 and 0.01 are applied in separate simulations.
+
+By default, there are 10 vertical layers. The sigma vertical coordinate is
+used. Horizontal resolution of 250m and 1 km are available.
+
+The test group includes 1 test case.  The test case has 4 steps,
+``initial_state``, which defines the mesh and initial conditions for the model,
+two ``forward`` steps, one for each Rayleigh damping coefficient, and a ``viz``
+which performs visualization of the ssh field and comparison with the analytical
+solution as well as the solution from the ROMS model.
+
+.. image:: images/drying_slope_validation.png
+   :width: 500 px
+   :align: center
+
+
+config options
+--------------
+
+The config options for this test case are:
+
+.. code-block:: cfg
+
+   # Options related to the vertical grid
+   [vertical_grid]
+   
+   # Number of vertical levels
+   vert_levels = 10
+   
+   # The type of vertical coordinate (e.g. z-level, z-star)
+   coord_type = sigma
+   
+   # config options for drying slope test cases
+   [drying_slope]
+   
+   # the number of grid cells in x and y
+   nx = 6
+   ny = 28
+   
+   # the size of grid cells (m)
+   dc = 1000.0
+   
+   # config options for visualizing drying slope ouptut
+   [drying_slope_viz]
+   
+   # frames per second for movies
+   frames_per_second = 30
+   
+   # movie format
+   movie_format = mp4
```

Comment:
Should this be updated with the changes to the config file?

Reply:
Good catch. I made those updates. Thanks, @sbrus89!